scholia/app/templates/base.html

```diff
@@ -62,7 +62,21 @@
 	     $("#h1").text(item.labels.en.value);
 	 }
 	 $("#h1").append(' (<a href="https://www.wikidata.org/wiki/{{ q }}">{{ q }}</a>)');
-	 
+
+    {% if q2 %}
+
+    var url2 = 'https://www.wikidata.org/w/api.php?action=wbgetentities&ids=' +
```

Comment:
Just adding a note, because it may conflict with https://github.com/WDscholia/scholia/pull/1540

Reply:
Happy for this PR to be merged first